pkg/transform/kubernetes/kubernetes_test.go

```diff
@@ -688,7 +688,7 @@ func TestTransformer_CreateDeployments(t *testing.T) {
 			}
 
 			if !reflect.DeepEqual(kd, test.K8sDeployments) {
-				t.Errorf("Expected: %#v\nGot: %#v\n", test.K8sDeployments, kd)
+				t.Errorf("Expected: %#v\nGot: %#v\n", test.K8sDeployments[0], kd[0])
```

Comment:
hardcoding the index values is not very good, use something like `spew.Sprint` to get the string format of the entire object.

I have done something similar in my PR https://github.com/redhat-developer/opencompose/pull/143 at https://github.com/redhat-developer/opencompose/pull/143/files#diff-c04853213c3b672346e99815d4cf3e55R760